src/timeline.rkt

```diff
@@ -198,6 +198,11 @@
 (define-timeline egraph #:unmergable)
 (define-timeline stop [reason false] [count +])
 (define-timeline branch #:unmergable)
+(define-timeline explanations [op false] [expr (const #f)] [expl false] [count +] [mcount +] [flows (const #f)])
+(define-timeline confusion #:custom (λ (x y) (list (map + (car x) (car y)))))
+(define-timeline total-confusion #:custom (λ (x y) (list (map + (car x) (car y)))))
+(define-timeline maybe-confusion #:custom (λ (x y) (list (map + (car x) (car y)))))
+(define-timeline freqs [key false] [val +])
```

Comment:
TBH I think having fewer of these would be good but I don't think this needs to hold up the merge.